pubsub/streaming-analytics/README.md

```diff
@@ -0,0 +1,171 @@
+# Stream Cloud Pub/Sub with Cloud Dataflow
+
+Sample(s) showing how to use [Google Cloud Pub/Sub] with [Google Cloud Dataflow].
+
+## Before you begin
```

Comment:
You can now omit all these verbose instructions. I've (recently) merged them into the [dataflow/README.md](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/dataflow/README.md). You can follow a similar format to this [example](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/dataflow/encryption-keys/README.md).

You might need to enable some extra APIs (like PubSub) and keep some of the sample-specific instructions not covered in that guide (like the cron publisher).

Reply:
Thanks for the link! Will use a similar format.